src/rhtap/postcreation/strategies/jenkinsPostCreateActionStrategy.ts

```
@@ -31,6 +32,9 @@ export class JenkinsPostCreateActionStrategy implements PostCreateActionStrategy
         new AddJenkinsSecretsCommand(component),
         new CreateJenkinsJobsCommand(component),
         new CreateWebhookCommand(component),
+        //First run must be triggered manually:
+        // https://stackoverflow.com/questions/56714213/jenkins-not-triggered-by-github-webhook#comment109322558_60625199 
```

ESLint

[error] Delete `·`

(prettier/prettier)